packages/cli/src/services/url.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Service } from 'typedi';
+import config from '@/config';
+
+@Service()
+export class UrlService {
+	/** Returns the base URL n8n is reachable from */
+	readonly baseUrl: string;
+
+	constructor() {
```

Comment:
Let's inject `config`?

Reply:
unfortunately injecting `config` has been tricky so far. I can try using a `Token` and `@Inject` in a separate PR for all services that use `config`.